pkgs/by-name/nh/nh/package.nix

```
@@ -7,14 +7,13 @@
   fetchFromGitHub,
   nix-update-script,
   nvd,
+  # Make optional as nom is AGPL. This mirrors nix-community/nh/package.nix
+  use-nom ? true,
```

```suggestion
  useNom ? true,
```

We usually use camel case for options

---

Hi- I was mirroring what was in https://github.com/nix-community/nh/blob/master/package.nix

What's the preference here?